ui/src/app/kvm/views/KVMDetails/KVMSummary/KVMSummary.tsx

```diff
@@ -82,7 +84,11 @@ const KVMSummary = (): JSX.Element => {
   const pod = useSelector((state: RootState) =>
     podSelectors.getById(state, Number(id))
   );
-  const [viewByNuma, setViewByNuma] = useState(false);
+  const [viewByNuma, setViewByNuma] = useStorageState(
```

Comment:
Is `viewByNuma` required?

Reply:
Yeah because it determines whether the aggregate or NUMA view is shown.